bigbluebutton-html5/private/config/settings.yml

```
@@ -743,6 +743,8 @@ public:
       enabled: true
       showNames: true
     moderatorChatEmphasized: true
+    privateMessageReadFeedback:
+      enable: false
```

```suggestion
      enabled: false
```

The config is different from the expected in the client code!